types/youtube-dl/youtube-dl-tests.tsx

```diff
@@ -0,0 +1,18 @@
+import * as fs from 'fs';
+import youtubedl = require('youtube-dl');
+
+const video = youtubedl('http://www.youtube.com/watch?v=90AiXO1pAiA',
+  // Optional arguments passed to youtube-dl.
+  ['--format=18'],
+  // Additional options can be given for calling `child_process.execFile()`.
+  { cwd: __dirname });
+
+// Will be called when the download starts.
+video.on('info', (info) => {
+  console.log('Download started');
```

Comment:
`console.log` is a poor method for testing types since it accepts `any`. You could just assert `info: youtubedl.Info` and that would be sufficient.

Reply:
Thank you for tip. I took example from npm package https://www.npmjs.com/package/youtube-dl. Library author use `console`. Do I have to change it here?

Reply:
Yes, you should try using the value in a way that actually requires it to be of its type. `console.log` takes `any` and can't fail no matter what the type of its argument is, which makes it useless for testing a type definition.

Reply:
@andy-ms . I'm not sure I got you right. Can you check out new updates?

Reply:
I'll have to do a `no-console` lint pass later. For now this should be fine.